api/ls_ganap/main_events/models.py

```diff
@@ -102,14 +102,11 @@ class Event(SoftDeletionModel):
 	created_at = models.DateTimeField(auto_now_add=True)
 	updated_at = models.DateTimeField(auto_now=True)
 	is_accepted = models.BooleanField(default=False)
-	poster_url = models.URLField()
+	poster_url = CloudinaryField('image')
```

Comment:
![screen shot 2018-08-10 at 10 37 55 pm](https://user-images.githubusercontent.com/20733806/43964252-fdddd2f4-9cee-11e8-8c08-1b06c465accf.png)

add poster_url to the event_serializer so it shows up in the json object